lobsterpy/cohp/test/test_describe.py

```diff
@@ -400,78 +426,129 @@ def test_text(self):
             ],
         )
         self.assertEqual(
             self.describe_CsH_all.text,
             [
                 "The compound CsH has 2 symmetry-independent atoms(s) with relevant bonds: Cs1, H2.",
-                "Cs1 has a 18-fold coordination environment. It has 18 Cs-Cs (mean ICOHP: -0.49 eV, 18.741 percent antibonding interaction below EFermi) bonds.",
+                "Cs1 has a 18-fold coordination environment. It has 18 Cs-Cs (mean ICOHP: -0.49 eV, 18.741 "
+                "percent antibonding interaction below EFermi) bonds.",
                 "H2 has a 0 coordination environment. It has 0 bonds.",
             ],
         )
+        self.assertEqual(
+            self.describe_BaTiO3_orb.text,
+            [
+                "The compound BaTiO3 has 1 symmetry-independent cation(s) with relevant cation-anion interactions: Ti2.",
+                "Ti2 has an octahedral (CN=6) coordination environment. It has 6 Ti-O (mean ICOHP: -3.54 eV, "
+                "1.092 percent antibonding interaction below EFermi) bonds.",
+                "The orbitals 2s-4s, 2s-3dxy, 2py-3dxy, 2px-3dx2, 2px-3dxy, 2py-3dyz, 2s-3dz2, 2pz-3dz2, "
+                "2px-3dxz, 2pz-4s contribute 21.79, 11.4, 13.1, 10.76, 13.6, 14.56, 15.26, 25.5, 13.75, "
+                "11.5 percent in the Ti-O bond, respectively.",
+            ],
+        )
+        self.assertEqual(
+            self.describe_NaSbF6_orb.text,
+            [
+                "The compound NaSbF6 has 3 symmetry-independent atoms(s) with relevant bonds: Na1, Sb2, F3.",
+                "Na1 has an octahedral (CN=6) coordination environment. It has 6 Na-F (mean ICOHP: -0.61 eV, "
+                "4.071 percent antibonding interaction below EFermi) bonds.",
+                "The orbitals 2s-3s, 2pz-3s, 2py-3s, 2px-3s contribute 77.48, 22.05, 22.05, "
+                "22.05 percent in the Na-F bond, respectively.",
+                "Sb2 has an octahedral (CN=6) coordination environment. It has 6 Sb-F (mean ICOHP: -5.45 eV, "
+                "0.0 percent antibonding interaction below EFermi) bonds.",
+                "The orbitals 2pz-5s, 2s-5pz, 2pz-5pz, 2py-5s, 2s-5py, 2py-5py, 2px-5s, 2s-5px, 2px-5px "
+                "contribute 19.49, 39.12, 21.41, 19.49, 39.12, 21.41, 19.49, 39.12, 21.41 percent in the Sb-F bond, "
+                "respectively.",
+                "F3 has a linear (CN=2) coordination environment. It has 1 F-Na (mean ICOHP: -0.61 eV, "
+                "4.545 percent antibonding interaction below EFermi), and 1 F-Sb (mean ICOHP: -5.45 eV, "
+                "0.0 percent antibonding interaction below EFermi) bonds.",
+                "The orbitals 2s-3s, 2pz-3s contribute 77.48, 22.05 percent in the F-Na bond, respectively., "
+                "and The orbitals 2pz-5s, 2s-5pz, 2pz-5pz contribute 19.49, 39.12, 21.41 percent in "
+                "the F-Sb bond, respectively.",
+            ],
```

Comment:
The order of orbitals in contrast to the bond randomly switches.

Reply:
Fixed it